components/org.wso2.carbon.identity.api.server.organization.user.invitation.management/org.wso2.carbon.identity.api.server.organization.user.invitation.management.v1/src/main/resources/organization-user-invitation.yaml

```diff
@@ -296,13 +296,14 @@ components:
     InvitationRequestBody:
       type: object
       required:
-        - username
+        - usernames
       properties:
-        username:
-          type: string
-          description: Username of the user who will be invited to the organization. This can be an
-            email or an alphanumeric username.
-          example: alex@gmail.com/alex
+        usernames:
+          type: array
+          description: List of username of the user's who will be invited to the organization. This can be an email or an alphanumeric username.
```

Comment:
```suggestion
          description: List of usernames of the users who will be invited to the organization. This can be an email or an alphanumeric username.
```